examples/image_ops.ipynb

```diff
@@ -3,88 +3,193 @@
   "nbformat_minor": 0,
```

Comment:
<p>Would you mind making this <code>pip install -q</code>. I have been meaning to update the other notebooks as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>